### PR TITLE
Contact numbers change success page

### PR DESCRIPTION
**Pull request by Jxr227**

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -107,6 +107,15 @@ emailChangeSuccess.helpTwo = Ensure your contact details are up to date.
 emailChangeSuccess.helpOne.digitalPreference = We will send you an email within 2 working days with an update, followed by a letter to your principal place of business. You can also go to your HMRC secure messages to find out if your request has been accepted.
 emailChangeSuccess.helpOne.paperPreference = We will send a letter to your principal place of business with an update within 15 working days.
 
+contactNumbersChangeSuccess.title = We have received the request to change your telephone numbers
```
Comment (lewishay) [inline conf/messages]: On the build version I see the heading/title here as "Request to change telephone numbers received".

File: test/controllers/contactNumbers/ContactNumbersChangeSuccessControllerSpec.scala [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+ */
+
+package controllers.contactNumbers
+
+import assets.CustomerInfoConstants.fullCustomerInfoModel
+import common.SessionKeys.{clientVrn, phoneNumberChangeSuccessful}
+import controllers.ControllerBaseSpec
+import mocks.MockContactPreferenceService
+import models.contactPreferences.ContactPreference
+import models.customerInformation.CustomerInformation
```
Comment (lewishay) [inline test/controllers/contactNumbers/ContactNumbersChangeSuccessControllerSpec.scala]: Unused import

File: app/controllers/contactNumbers/ContactNumbersChangeSuccessController.scala [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+    if (user.session.get(phoneNumberChangeSuccessful).exists(_.equals("true"))) {
+      for {
+        customerDetails <- if (user.isAgent) {vatSubscriptionService.getCustomerInfo(user.vrn)}
+                           else {Future.successful(Left(ErrorModel(NO_CONTENT, "")))}
+        preference <- if (user.isAgent) {Future.successful(Left(ErrorModel(NO_CONTENT, "")))}
+                      else {contactPreferenceService.getContactPreference(user.vrn)}
+      } yield {
+        val viewModel = constructViewModel(customerDetails, preference, user.session.get(verifiedAgentEmail))
+        Ok(contactNumberChangeSuccessView(viewModel)).removingFromSession(prepopulationLandlineKey, prepopulationMobileKey)
```
Comment (lewishay) [inline app/controllers/contactNumbers/ContactNumbersChangeSuccessController.scala]: These two session keys would have already been removed as part of the `updateContactNumbers` function in `ConfirmContactNumbersController`

File: test/views/contactNumbers/ContactNumbersChangeSuccessViewSpec.scala [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+        "have the correct second paragraph" in {
+          elementText(Selectors.paragraphTwo) shouldBe "We‘ll also contact TheBusiness with an update."
+        }
+
+        "have a GA tag for the clicking finish button" in {
+          element(Selectors.button).select("h1").attr("data-journey") contains "email-address:confirm:finish-email-change"
+        }
+      }
+
+      "the agent doesn't have an email address registered" should {
```
Comment (lewishay) [inline test/views/contactNumbers/ContactNumbersChangeSuccessViewSpec.scala]: Could you add in an extra test case just for the small content change when the agent's client's business name is not retrieved

File: conf/messages (hunk 1 above)
Comment (Jxr227) [reply conf/messages]: You're right, could've sworn I saw this wording somewhere but I must've either looked at a different journey or imagined it.

File: test/views/contactNumbers/ContactNumbersChangeSuccessViewSpec.scala (hunk 4 above)
Comment (Jxr227) [reply test/views/contactNumbers/ContactNumbersChangeSuccessViewSpec.scala]: Ahh assumed the helper was unit tested so wouldn't need to. On it now

**Pull request merged**